src/murthy_lab_to_nwb/cowley2022mapping/cowley2022mappingaudiointerface.py

```diff
@@ -46,9 +46,8 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
         acoustic_waveform_series = AcousticWaveformSeries(
             name="audio_waveforms",
             data=audio_data,
-            rate=30.0,  # TODO confirm with authors
+            rate=10.0,
```

Comment:
a sampling rate of 10 Hz for a sound waveform? That' very low.

Reply:
They did mention that they downsampled it after recording